template.yaml

```diff
@@ -151,11 +141,8 @@ Resources:
       Handler: no.unit.alma.GetAlmaSruRecordHandler::handleRequest
       Runtime: java11
       MemorySize: 2048
-      AutoPublishAlias: live
       DeploymentPreference:
         Type: AllAtOnce # Or Canary10Percent5Minutes, Linear10PercentEvery1Minute, ...
```

Comment:
You must specify an AutoPublishAlias in your AWS::Serverless::Function to use a DeploymentPreference object, otherwise an error will result.